modules/light-clients/08-wasm/types/config.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+const (
+	// contractMemoryLimit is the memory limit of each contract execution (in MiB)
+	// constant value so all nodes run with the same limit.
+	ContractMemoryLimit = 32
+
+	defaultDataDir           string = "ibc_08-wasm_client_data"
```

Comment:
This is the default data dir if 08-wasm is used standalone, correct?

Reply:
yeah.